src/gen/async_generate_data_gemini.py

```diff
@@ -0,0 +1,99 @@
+import asyncio
+import aiohttp
+from typing import List, Dict
+import os
+import sys
+import json
+import numpy as np
+from datasets import load_dataset
+
+
+if os.getenv("COLAB_RELEASE_TAG"):
+    from google.colab import userdata
+
+    GOOGLE_API_KEY = userdata.get("GEMINI_ONE_API_KEY", None)
+else:
+    GOOGLE_API_KEY = os.getenv("GEMINI_ONE_API_KEY", None)
+
+assert GOOGLE_API_KEY, "GOOGLE_API_KEY cannot be None."
+
+
+MODEL_NAME = "gemini-1.0-ultra-latest"
+NUM_DATASET_SAMPLES = 5
+EXISTING_DATASET = load_dataset("sayakpaul/no_robots_only_coding", split="train_sft")
+RATE_LIMIT_PER_MINUTE = 60
+REQUEST_INTERVAL = 60 / RATE_LIMIT_PER_MINUTE  # Seconds between requests
+
+
+# prompt template can be loaded from a `.toml` file.
+def craft_prompt(instruction, response):
```

Comment:
The prompt shouldn't be hard coded/embeded into the Python script itself. How about adding this into the `toml` file inside `configs` directory?

Reply:
https://github.com/deep-diver/llmops-pipeline/pull/10#discussion_r1546067182